android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TiTableViewRowProxyItem.java

```diff
@@ -43,7 +43,7 @@
 	// every time we add a row. No sense checking it each time.
 	private static boolean ICS_OR_GREATER = (Build.VERSION.SDK_INT >= TiC.API_LEVEL_ICE_CREAM_SANDWICH);
 
-	private static final int LEFT_MARGIN = 5;
+	private static final int LEFT_MARGIN = 22;
```

Comment:
A bit hard to find out as I find various examples (from 0 margin to 48). But I found this theme.xml:
https://github.com/android/platform_frameworks_base/blob/82834baa358f55acb542e17da828b2d497cf8332/core/res/res/values/themes.xml#L146
and it looks like 6dp is a default value for listitems in a basic theme. I will update ListView and TableView to look the same